lib/game_play.rb

```diff
@@ -13,29 +13,47 @@ class GamePlay
   def initialize(ui)
     @board = Board.new
     @ui = ui
+    @VALUE_MIN = 1
   end
 
   def board_size
-    @ui.board_size
-    selection = get_user_selection_between(3, 4)
+    default_size = 1
+
+    @ui.display_menu_board(default_size)
+    menu_size = @ui.menu_board_size
+    selection = get_user_value(@VALUE_MIN, menu_size, default_size)
+
+    selection += 2
```

Comment:
Adding 2 seems like magic to me.  You have some knowledge that is unclear to me why it should be 2.  Could you envision this ever changing?